server/config/routes.rb

```diff
@@ -1,8 +1,7 @@
 Rails.application.routes.draw do
   post "/api/graphql", to: "graphql#execute"
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  
-  if Rails.env.development?
-    mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/api/graphql"
-  end
+
+  get '/health_check', to: proc { [200, {}, ['success']] } unless Gene.first.nil?
+
+  mount GraphiQL::Rails::Engine, at: "/graphiql", graphql_path: "/api/graphql"
```

Comment:
I don't know how you've got everything set up on beanstalk so, totally up to you guys, but I tend to put this under `/api/graphiql` in production just because that makes the logic easier for whatever your reverse proxy config setup is. You can just tell it anything at `/api` should be sent to puma and anything else should be served as a static asset.